dev_apps/music-nga/views/home/view.js

```diff
@@ -80,4 +81,13 @@ HomeView.prototype.queueAlbum = function(filePath) {
   this.fetch('/api/queue/album/' + filePath);
 };
 
+HomeView.prototype.search = function(query) {
+  // XXX: Search all fields
+  return this.fetch('/api/search/title/' + query).then((response) => {
```

Comment:
Maybe we just have a fake "all" key that internally does a search on all real keys.